qcfractal/queue/managers.py

```diff
@@ -468,6 +473,7 @@ def test(self, n=1) -> bool:
 
         if failures:
             self.logger.error("{}/{} tasks failed!".format(failures, len(results)))
+            self.logger.error(f"Last known error to help you debug it:\n{result.error.error_message}")
```

Comment:
Makes me a little uncomfortable view a iterator depends, it also doesn't work for multiple programs in the test.